src/walkingDownStairs.ts

```diff
@@ -51,21 +50,12 @@ export class WalkDownStairs {
         this.stage0Config = { ...this.config };
         this.container = document.getElementById(id);
         this.createCanvas();
-        this.gameStore.fpsObj = new LMS_Text({
-            text: '', x: this.canvas.width - 60, y: 20, font: '14px sans-serif',
-            color: '#0095DD', align: 'left',
-        });
+        this.gameStore.fpsObj = ShapeGeneratorHelper.getLMSText('', this.canvas.width - 60, 20, '14px sans-serif', '#0095DD', 'left');
```

Comment:
same

Reply:
Same as above for src/shape/status-shape.ts